src/components/payments/splits/splitsColumns.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+
+import { ColumnDef } from "@tanstack/react-table";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { truncateString } from "@/app/(dashboard)/utils/stringUtils";
+import { SplitRecipient } from "./splitsRecipients";
+
+export const SplitsColumns: ColumnDef<SplitRecipient>[] = [
+  {
+    id: "avatar",
+    cell: ({ row }) => {
+      const data = row.original;
+      return (
+        <div>
+          {data.username ? (
+            <Avatar className="h-8 w-8">
+              <AvatarImage
+                src={`https://github.com/${data.username}.png?size=100`}
+                alt={data.address}
+              />
+              <AvatarFallback>{data.address}</AvatarFallback>
+            </Avatar>
+          ) : (
+            <Avatar className="h-8 w-8">
+              <AvatarImage
+                src={`https://avatars.jakerunzer.com/${data.address}`}
+                alt={data.address}
+              />
+              <AvatarFallback>{data.address}</AvatarFallback>
+            </Avatar>
+          )}
+        </div>
+      );
+    },
+  },
+  {
+    accessorKey: "wallet",
```

Comment:
Let's have a hardcoded check for Armitage wallet here, so that we can display it as armitage

Reply:
good idea